src/main/groovy/org/transmartproject/batch/gwas/metadata/GwasAnalysisPartitioner.groovy

```diff
@@ -0,0 +1,34 @@
+package org.transmartproject.batch.gwas.metadata
+
+import groovy.util.logging.Slf4j
+import org.springframework.batch.core.partition.support.Partitioner
+import org.springframework.batch.item.ExecutionContext
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+/**
+ * Creates the partitions for splitting the flow of the job for each analysis
+ */
+@Component
+@Slf4j
+class GwasAnalysisPartitioner implements Partitioner {
```

Comment:
Ah this is a `Partitioner` in the Spring Batch sense (how a workload should be split across workers), not in an RDBMS sense.